### PR TITLE
feat: improve logging for detect command

### PR DESCRIPTION
**Pull request by kvendingoldo**

### CODE REVIEW

**Comment by kvendingoldo** [comment]
related to https://github.com/tofuutils/tenv/issues/163

**Comment by kvendingoldo** [comment]
@dvaumoron any comments ? :)

File: versionmanager/manager.go [hunk 1]
```diff
@@ -112,12 +114,22 @@ func (m VersionManager) Evaluate(requestedVersion string, proxyCall bool) (strin
 			}
 		}
 
+		m.conf.Displayer.Display("No compatible version found locally, search a remote one...")
 		if m.conf.NoInstall {
 			m.conf.Displayer.Flush(proxyCall)
```
Comment (dvaumoron) [inline versionmanager/manager.go:119]: This line is no longer needed

File: versionmanager/manager.go [hunk 2]
```diff
@@ -112,12 +114,22 @@ func (m VersionManager) Evaluate(requestedVersion string, proxyCall bool) (strin
...
 
-			return "", errNoCompatible
+			version, err := m.searchInstallRemote(predicateInfo, m.conf.NoInstall, proxyCall)
+			if err != nil {
+				return "", errNoCompatibleLocally
+			}
+
+			cmdName := strings.ToLower(m.FolderName)
+
+			m.conf.Displayer.Display("Auto-install is disabled. To install " + version + " version you can set environment variable TENV_AUTO_INSTALL=true, or install it via any of the following command: 'tenv " + cmdName + " install', 'tenv " + cmdName + " install " + version + "'")
```
Comment (dvaumoron) [inline versionmanager/manager.go:128]: You should use `loghelper.Concat`, instead of multiple `+` (more efficient as it use a `strings.Builder`)

**Review by dvaumoron — CHANGES_REQUESTED** [review]

File: versionmanager/manager.go (hunk 1 above)
Comment (dvaumoron) [inline versionmanager/manager.go:119]: This line is not needed here anymore, however it should be called before `return "", errNoCompatibleLocally`

**Review by dvaumoron — CHANGES_REQUESTED** [review]

**Comment by kvendingoldo** [comment]
Updated

**Review by dvaumoron — APPROVED** [review]
LGTM

**Pull request merged**